contrib/builder/deb/armhf/generate.sh

```diff
@@ -83,15 +83,15 @@ for version in "${versions[@]}"; do
 	)
 	# packaging for "sd-journal.h" and libraries varies
 	case "$suite" in
-		precise|wheezy) ;;
-		jessie|trusty) packages+=( libsystemd-journal-dev );;
-		*) packages+=( libsystemd-dev );;
+		wheezy) ;;
+		jessie|trusty) packages+=( libsystemd-journal-dev ) ;;
+		*) packages+=( libsystemd-dev ) ;;
```

Comment:
@thaJeztah It's perfect with your change. I'm not sure if the added blank is actually required, but let's make your editor happy.